[docs] Update using-fcm.md (#18583)
expo/expo: docs/pages/push-notifications/using-fcm.md

2. In your new project console, click **Add Firebase to your Android app** and follow the setup steps. **Make sure that the Android package name you enter is the same as the value of `android.package` in your app.json.**

3. Download the `google-services.json` file and place it in your Expo app's root directory.

>   > **Note:** The Firebase config file contains unique, but non-secret identifiers for your project. To learn more about this config file, visit [Understand Firebase Projects](https://firebase.google.com/docs/projects/learn-more#config-files-objects).

Reviewer: ```suggestion
  > **Note:** The `google-services.json` file contains unique and non-secret identifiers of your Firebase project. For more information, see [Understand Firebase Projects](https://firebase.google.com/docs/projects/learn-more#config-files-objects).
```
I think the file name should be mentioned explicitly. Also, need to be explicit about the config file containing info about Firebase project and not Expo project. In our docs, we usually refer to "projects" in individuality as Expo projects.
